apps/pie-docs/src/componentStatusData.js

```diff
@@ -1435,7 +1435,8 @@ const rows = [
         },
         {
             resource: resourceTypes.WEB_COMPONENTS,
-            status: statusTypes.PLANNED,
+            link: 'https://webc.pie.design/?path=/story/radio--default',
```

Comment:
Do we want a separate row in the status table for radio-group?

Reply:
I think this was a mistype from me. It should be radio-group. Updating now